### PR TITLE
chore: tee to lily.log

### PR DESCRIPTION
**Pull request by kasteph**

### CODE REVIEW

File: scripts/export.sh [hunk 1]
```diff
@@ -35,7 +35,7 @@ echo "Running daemon..."
 lily daemon \
     --repo="${REPO_PATH}" \
     --config=/lily/config.toml \
-    --bootstrap=false &> lily.log &
+    --bootstrap=false > >(tee lily.log) 2&>1 &
```
Comment (birdychang) [inline scripts/export.sh]: This doesn't work. You need to do `&> >(tee lily.log) &`

**Review by birdychang — CHANGES_REQUESTED** [review]

File: scripts/export.sh (hunk 1 above)
Comment (kasteph) [reply scripts/export.sh]: thanks, I changed it

**Review by birdychang — APPROVED** [review]

**Comment by davidgasquez** [comment]
Already discussed it with @birdychang but wanted to leave it here too. Outputing all Lily logs to STDOUT/STERR will cause a large spike on Cloud Logging billing.

This is how the last time I tried looked like. Green is Cloud Logging.



It goes from 30% to 50% of the total costs. That said, perhaps we could only log `warn`/`error` level stuff.

**Comment by kasteph** [comment]
> Already discussed it with @birdychang but wanted to leave it here too. Outputing all Lily logs to STDOUT/STERR will cause a large spike on Cloud Logging billing.

Sorry, I wasn't aware of that. We needed to enable it in order to debug the remaining issues, even after the job has disappeared. Feel free to toggle the logs.

**Comment by davidgasquez** [comment]
Yeah, is a hard tradeoff. All issues that Lily can handle should be already appearing in the STDOUT. The only one that's not showing is #22 due the daemon being killed and bringing the entire job with it.

Since we're running things manually, we can leave it on for a while

**Pull request merged**